tests/test_core.py

```diff
@@ -7523,7 +7523,7 @@ def test_asyncify_lists(self, args, should_pass, response=None):
 
     # use of ASYNCIFY_* options may require intermediate debug info. that should
     # not end up emitted in the final binary
-    if self.is_wasm():
+    if self.is_wasm() and not is_sanitizing(self.emcc_args):
```

Comment:
Why this?

Reply:
The code under this looks for "name" in the binary. ASan builds always contain due to all the static strings they include... I'll add a comment.

Reply:
Sounds like we should have better check for the data section instead?

Reply:
Ideally, but parsing the wasm binary seems like overkill here? It would not be the end of the world if an asan build included the names section by mistake even if this regressed somehow.